packages/shared/lib/services/sync/run.service.ts

```diff
@@ -611,27 +611,31 @@ export default class SyncRun {
 
     async finishFlow(models: string[], syncStartDate: Date, version: string, totalRunTime: number, trackDeletes?: boolean): Promise<void> {
         let i = 0;
-        for (const model of models) {
-            let deletedKeys: string[] = [];
-            if (!this.isWebhook && trackDeletes) {
-                deletedKeys = await this.recordsService.markNonCurrentGenerationRecordsAsDeleted({
-                    connectionId: this.nangoConnection.id as number,
+
+        if (!this.isAction && !this.isWebhook && !this.isPostConnectionScript) {
```

Comment:
no `isSync`?

Reply:
does not exist, I didn't wanted to start a refacto now :D